ludwig/encoders/sequence_encoders.py

```diff
@@ -795,12 +792,11 @@ def output_shape(self) -> torch.Size:
             return self.conv1d_stack.output_shape
         return self.fc_stack.output_shape
 
-    def forward(self, inputs, mask=None):
-        # todo: fixup docstring
+    def forward(self, inputs: torch.Tensor, mask: Optional[torch.Tensor] = None):
         """
-        :param input_sequence: The input sequence fed into the encoder.
+        :param inputs: The input sequence fed into the encoder.
                Shape: [batch x sequence length], type torch.int32
-        :type input_sequence: Tensor
+        :param mask: Input mask (unused, not yet implemented)
```

Comment:
This is a good comment. This might be good to discuss at our next sync since `mask` seems to be passed but unused/unimplemented in many places.